programs/ssl/ssl_mail_client.c

```diff
@@ -57,8 +57,10 @@
 #include <unistd.h>
 #else
 #include <io.h>
-#define read _read
-#define write _write
+#define read(fd, buf, len)                                      \
+    _read( fd, (void *)( buf ), (unsigned int)( len ) )
```

Comment:
Yes, we should raise an issue for this, but if possible, fis it here while we are modifying this file